CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.56.0]
+
+### Breaking Changes
+
+* Remove obsolete linker options (`--mir-linker` and `--legacy-linker`) by @zhassan-aws in https://github.com/model-checking/kani/pull/3559
+
+### What's Changed
```

Comment:
We should also mention that `kani::check` is deprecated (https://github.com/model-checking/kani/pull/3557).